source/interfaces/GitHistory.gd

```diff
@@ -4,20 +4,27 @@ export (PackedScene) var git_command = preload("res://interfaces/GitCommand.tscn
 
 func add_git_command(text):
 	if get_child_count() > 4:
-		var first = get_child(0)
-		first.queue_free()
+		for child in get_children():
+			if not child.is_queued_for_deletion():
+				child.queue_free()
+				break
 	var instance = git_command.instance()
 	instance.text = text
 	add_child(instance)
 
 
 func _on_player_head_area_entered(area):
 	if area.is_in_group("bug"):
+		PlayerData.commit_counters['fix'] += 1
 		add_git_command("git merge bug-fix")
 	elif area.is_in_group("feature"):
+		PlayerData.commit_counters['feature'] += 1
 		add_git_command('git commit -m "new feature!"')
 	elif area.is_in_group("release"):
 		add_git_command("git merge development")
+		PlayerData.bump_version()
+		var v = PlayerData.current_version
```

Comment:
even for jam entries I'm not a fan of 1 letter variables, this is supposed to be `version`. I will push a commit to fix this :sweat_smile:

Reply:
I use one letter variables everywhere, shame on me :v